src/files-folder-view.tsx

```diff
@@ -57,6 +57,8 @@ export const FilesFolderView = ({
               setIsGrid={setIsGrid}
               sortBy={sortBy}
               setSortBy={setSortBy}
+              files={files}
+              path={path}
```

Comment:
How's our context provider work coming along?

Reply:
Does not include `path` yet, I some code lying around for that.